Add link and quick instructions for maude (#3)
lumaier/securedrop-formalanalysis: README.md

Thus, you must compile Tamarin from source to construct or verify proofs.
To do so, follow the instructions here: https://tamarin-prover.com/install.html

> First, you must install Tamarin's dependencies, including [maude](https://maude.cs.illinois.edu/get-maude), and have it in PATH.

Reviewer: ```suggestion
First, you must install Tamarin's dependencies, including [maude](https://maude.cs.illinois.edu/get-maude), and have add it to your `PATH`.
```
